controllers/util/execPod_test.go

```diff
@@ -0,0 +1,285 @@
+package util
+
+import (
+	"bytes"
+	"context"
+	apiMocks "github.com/cloudogu/k8s-dogu-operator/api/v1/mocks"
+	"github.com/stretchr/testify/mock"
+	corev1 "k8s.io/api/core/v1"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/cloudogu/cesapp-lib/core"
+	"github.com/cloudogu/k8s-dogu-operator/controllers/resource"
+	"github.com/cloudogu/k8s-dogu-operator/controllers/util/mocks"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	k8sv1 "github.com/cloudogu/k8s-dogu-operator/api/v1"
+)
+
+const testNamespace = "ecosystem"
+const podName = "test-execpod-123abc"
+const containerName = "ldap"
+
+var testCtx = context.TODO()
+
+func Test_defaultSufficeGenerator_String(t *testing.T) {
+	actual := (&defaultSufficeGenerator{}).String(6)
+	assert.Len(t, actual, 6)
+}
+
+func TestExecPod_ObjectKey(t *testing.T) {
+	// given
+	inputResource := &k8sv1.Dogu{
+		ObjectMeta: metav1.ObjectMeta{Name: "le-dogu", Namespace: testNamespace},
+	}
+	sut := &execPod{podName: podName, doguResource: inputResource}
+
+	// when
+	actual := sut.ObjectKey()
+
+	// then
+	assert.NotEmpty(t, actual)
+	expected := &client.ObjectKey{
+		Namespace: testNamespace,
+		Name:      podName,
+	}
+	assert.Equal(t, expected, actual)
+}
+
+func Test_execPod_Create(t *testing.T) {
```

Comment:
Assert Expectations from the mocks

Reply:
fixed by using the mockery-generated constructor